docker/stacks/development/docker-compose.yml

```diff
@@ -1,10 +1,10 @@
-version: "3.7"
+version: "3.9"
```

Comment:
+1 if we give this an explicit name like `name: parabol-dev` then the project name won't change when the folder changes.

Reply:
I'll add `name: parabol-dev` tomorrow morning before merging! Good idea!